README.md

```diff
@@ -1,23 +1,129 @@
 
-# H3lp!
+<h1 align="center">
+  H3LP!
+ <br>
+  <a href="https://github.com/H3LP3RS/HELP"><img src="https://github.com/H3LP3RS/HELP/blob/main/app/src/main/res/drawable/heart_link.png" width="300"></a>
+  <br>
+</h1>
+
+<h2 align="center">Anyone can be an hero.</h2>
+
+<p align="center">
+
+  <a href="https://github.com/H3LP3RS/HELP/commit/main">
+    <img src="https://img.shields.io/github/last-commit/H3LP3RS/HELP"
+        alt="Last commit">
+  </a>
+
+  <a href="https://github.com/H3LP3RS/HELP/graphs/contributors">
+      <img src="https://img.shields.io/github/contributors/H3LP3RS/HELP"
+       alt="Contributors">
+  </a>
+
+ 
  
- This android application is developed for the project course  [CS-306 Software Development Project ](https://dslab.epfl.ch/teaching/sweng/proj) at  EPFL. 
+  <a href="https://cirrus-ci.com/github/H3LP3RS/HELP">
+    <img src="https://api.cirrus-ci.com/github/H3LP3RS/HELP.svg"
+         alt="Build Status">
+  </a>
+    
+  <a href="https://codeclimate.com/github/H3LP3RS/HELP/test_coverage">
+    <img src="https://api.codeclimate.com/v1/badges/dbecacba890747624c24/test_coverage"
+         alt="Test coverage">
+  </a>
+    
+  <a href="https://codeclimate.com/github/H3LP3RS/HELP/maintainability">
+    <img src="https://api.codeclimate.com/v1/badges/dbecacba890747624c24/maintainability"
+         alt="Mainttainability">
+  </a>
+    
+</p>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="https://www.youtube.com/watch?v=d6ds5Lch3GA">Presentation video</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#screenshots">Screenshots</a> •
+  <a href="#the-team">The team</a>
+</p>
 
-## Description
 The purpose of this app is to offer emergency help by allowing willing users to register and enter their skills (e.g. have taken courses on first aid) and the first aid medicine they usually walk around with (eg. EpiPens for people with severe allergies, asthma inhalers for people with asthma, antihistamines for allergies…).
 
 When a user suddenly needs their medication (e.g. if they had an anaphylactic shock), after calling an ambulance, them (or the people they are with) can go on the app and ask for a certain medication or for someone with a particular first aid skill, the app will then notify any matching users in a certain radius so that they can come and help.
 
+It was created as part of the Software Development Project [CS-306](https://edu.epfl.ch/coursebook/en/software-development-project-CS-306-1)
+class at EPFL.
+
+## Key Features
+
+Once the app is downloaded the users can sign in and fill up there information:
+- signe in with Google or anonymously. 
+- Fill up there medical informing, containing useful information in case of emergency like blood type, allergy or current Treatment.
+- Add emergency contact to call in case of emergency.
+- Add there medical skills and the medicine they carry with them.
+
```

Comment:
```suggestion
- Fill up their medical information, containing useful information in case of emergency like blood type, allergy or current Treatment.
- Add  an emergency contact to call in case of emergency.
- Add there medical skills and the medicine they carry with them.

```